.github/workflows/cleanup-terraform.yaml

```diff
@@ -0,0 +1,33 @@
+# Workflow that runs whenever a PR is closed or merged and deletes any
+# AWS resources created by the `deploy` workflow.
+name: cleanup-terraform
```

Comment:
See [this workflow run](https://github.com/ccao-data/model-res-avm/actions/runs/6711800427/job/18239982559) for a test of this workflow demonstrating successful completion.